apps/web/pages/organizations/[organizationId]/invoices/[invoiceId]/index.page.tsx

```diff
@@ -15,9 +15,12 @@ const InvoiceQueryDocument = graphql(`
 `)
 
 const InvoiceDetailsPage = (): JSX.Element => {
-  const context = useMemo(() => ({ additionalTypenames: ['Organization', 'InvoiceItem'] }), [])
   const router = useRouter()
   const { invoiceId, organizationId } = router.query
+  const context = useMemo(
+    () => ({ additionalTypenames: ['Organization', 'Invoice', 'InvoiceItem'] }),
```

Comment:
I don't think we need `Organization` or `Invoice` type here?
```suggestion
    () => ({ additionalTypenames: ['InvoiceItem'] }),
```